src/applications/vaos/components/cancel/CancelAppointmentConfirmationModal.jsx

```diff
@@ -22,6 +22,7 @@ export default function CancelAppointmentConfirmationModal({
       <p className="vads-u-margin-top--2">
         <LoadingButton
           isLoading={status === FETCH_STATUS.loading}
+          loadingText="Page change in progress"
```

Comment:
If this is the only text we're going to use, we could put it in FormButtons.jsx.

Reply:
I had it like that at 1st but thought FormButton seemed like a generic component. I can change though.

Reply:
Yeah, I think they're generic to the form pages, so "Page change in progress" is probably ok. Though we have some scenarios where those buttons are set to loading before a page change, so we'd still want the ability to customize it.